root/layout/header/bottom.tt

```diff
@@ -96,7 +102,7 @@
     </li>
     [%- IF c.user_exists -%]
         <li class="editing">
-            <a href="[% doc_link('How_Editing_Works') %]">[% l('Editing') %]</a>
+            <span class="menu-header">[% l('Editing') %]</span>
```

Comment:
We dropped the link to the voting guide, and this isn't much more useful - plus the guides are already linked under Documentation in the How-To page, where it'd make more sense for someone to look for them.